mftf/2.3/test/action-groups.md

```diff
@@ -142,29 +147,35 @@ For example, the following defines the replacement argument `relevantString` usi
 
 The `string` argument type provides a method to pass a single piece of data to the `<actionGroup>`during a test instead of passing an entire entity.
 
-**To explicitly define the argument value**:
+### Define the argument value
 
 ```xml
 <actionGroup stepKey="fillWithStringLiteral" ref="fillExample">
     <argument name="relevantString" value="overrideString"/>
 </actionGroup>
 ```
 
-**To define the argument value using persisted data references**:
+#### Using persisted data references
 
 ```xml
 <actionGroup stepKey="fillWithStringLiteral" ref="fillExample">
     <argument name="relevantString" value="$persistedData.field1$"/>
 </actionGroup>
 ```
 
-**To define the argument value based on data entity resolution**:
+The `relevantString` argument value points to the data [created](../data.html#persisting-a-data-entity-as-a-prerequisite-of-a-test){:target="_blank"} in the `stepKey="persistedData"` test step.
+`field1` is a data key of the required data string.
+Even with the `persistedData` data entity, the MFTF interprets the `$persistedData.field1$` value as a string.
+
+#### Based on data entity resolution
```

Comment:
Another H3, with a parallel name. Revert to the origin pseudo-heading, minus the "To"

Reply:
:ok: